cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCards.kt

```diff
@@ -0,0 +1,30 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+object VitaminCards {
+
+    /**
+     * The card is a flexible and extensible content container
+     * @param modifier The [Modifier] to be applied to the component
+     * @param colors The colors for the card, background of the card can be customized
+     * @param sizes The sizes for the card, elevation can be customized
+     * @param content The content to add inside the card
+     */
+    @Composable
+    fun Primary(
+        modifier: Modifier = Modifier,
+        colors: CardColors = VitaminCardsColors.primary(),
+        sizes: CardSizes = VitaminCardSizes.medium(),
+        content: @Composable () -> Unit
+    ) {
+        Card(
+            modifier = modifier,
+            backgroundColor = colors.backgroundColor,
```

Comment:
Here you can add `shape` properties to the Card

Reply:
ok